src/lib/realmdb/RealmDB.js

```diff
@@ -393,7 +393,8 @@ class RealmDB implements DB, ProfileDB {
         .filter(
```

Comment:
pls rework to use mongodb's `$in`
also apply corresponding filters according to the category passed

Reply:
@johnsmith-gooddollar this is not possible, see the comment above, that it uses dexie directly and not mongoify